--- nyquist/sys/unix/linux/Makefile.orig	2004-11-10 11:07:00.000000000 +0100
+++ nyquist/sys/unix/linux/Makefile	2006-05-24 13:32:23.000000000 +0200
@@ -5,11 +5,11 @@
 # DESC: This file builds various utilities for Nyquist
 #
 
-CC = gcc $(CFLAGS)
+CC = gcc
 
 # the unix path gets us switches.h:
 # the cmt path gets us swlogic.h:
-CFLAGS = -g -I../sys/unix -I../cmt
+CFLAGS = $(OPTFLAGS) -I../sys/unix -I../cmt
 
 # Directory info:
 BINPATH = .
--- nyquist/misc/Makefile.orig	2005-12-22 10:48:00.000000000 +0100
+++ nyquist/misc/Makefile	2006-05-24 13:31:23.000000000 +0200
@@ -41,7 +41,8 @@
 # to enable command line editing, use -DREADLINE. WARNING: THIS WILL DISABLE THE
 # ABILITY TO INTERRUPT LISP AND USE SOME OTHER HANDY CONTROL CHARACTERS
 CFLAGS = -DCMTSTUFF -DPA_LITTLE_ENDIAN -g $(INCL) -DHAVE_LIBPTHREAD=1 \
-     -DPA_USE_OSS=1 $(USE_ALSA) -DSTK_NYQUIST
+     -DPA_USE_OSS=1 $(USE_ALSA) -DSTK_NYQUIST $(OPTFLAGS)
+
 LN = g++
 # to disable command line editing, take out -lreadline -lcurses
 LFLAGS = -lm -lreadline -lcurses -lpthread -lasound
